airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -2354,6 +2354,10 @@ class AsyncRetriever(BaseModel):
         ...,
         description="Requester component that describes how to prepare HTTP requests to send to the source API to fetch the status of the running async job.",
     )
+    polling_timeout: Optional[Union[int, str]] = Field(
```

Comment:
Should we had `job` somewhere in this? With this name, my understand was that we were to stop all the polling after the timeout not the one from a specific job.

Reply:
This gets applied to all the jobs before we start creating them. This is the general setting applied to all the jobs. Each job will take this timeout into consideration. We don't have the possibility to control `some jobs` and `leave some of them run longer` do we?

Reply:
Oh, you mean `just rename it to `polling_job_timeout`` because this is job-related. I agree.

Reply:
This is what I meant, yes

Reply:
Done.